<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>268</width>
     <height>273</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QLineEdit" name="SendEdit"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="SendBtn">
      <property name="text">
       <string>发送信息</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Ip地址：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="PortEdit"/>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QTextEdit" name="GetText"/>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="IpEdit"/>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="btn">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
